### PR TITLE
added default value for complete_status

### PR DESCRIPTION
**Pull request by ziafazal**
@markoivanovic added default value for `complete_status`

### CODE REVIEW

**Comment by marjev** [comment]
@martynjames complete_status now has a default value. Can we merge this?

**Comment by martynjames** [comment]
:+1: from me!

**Comment by martynjames** [comment]
@mattdrayer - Feel comfortable merging this?

File: lms/djangoapps/organizations/views.py [hunk 1]
```diff
@@ -115,7 +115,7 @@ def users(self, request, pk):
                         enrollments = CourseEnrollment.enrollments_for_user(user).count()
                         user_data['course_count'] = enrollments
                     if str2bool(include_grades) and course_key:
-                        user_grades = {'grade': 0, 'proforma_grade': 0}
+                        user_grades = {'grade': 0, 'proforma_grade': 0, 'complete_status': False}
```
Comment (mattdrayer) [inline lms/djangoapps/organizations/views.py]: I guess I'm not up on the exact use case here -- why are we always returning False?

Comment (martynjames) [inline lms/djangoapps/organizations/views.py]: Shouldn't always be returning false, should be defaulting to false in case there is no gradebook record.

**Pull request merged**